src/dates/date.md

```diff
@@ -0,0 +1,209 @@
+
+# How to work with Dates
+## The Date API 
+
+As of `Java 8 SE` with the release of the [Date Class](https://docs.oracle.com/javase/8/docs/api/java/util/Date.html) defined as:
+> The class `Date` represents a specific instant in time, with millisecond precision.
+
+Also further on we had the addition of the [Abstract Calender Class](https://docs.oracle.com/javase/8/docs/api/java/util/Date.html) (with significant upgrade with JDK 1.1) defined as:
+> The `Calendar` class is an abstract class that provides methods for converting between a specific instant in time and a set of [`calendar fields`](https://docs.oracle.com/javase/8/docs/api/java/util/Calendar.html#fields) such as `YEAR`, `MONTH`, `DAY_OF_MONTH`, `HOUR`, and so on, and for manipulating the calendar fields, such as getting the date of the next week. An instant in time can be represented by a millisecond value that is an offset from the _Epoch_, January 1, 1970 00:00:00.000 GMT (Gregorian).
+
+As noted by the [Oracle docs](https://docs.oracle.com/javase/8/docs/api/java/util/Calendar.html) the reason was the lack of internationalization abilities.
+
+*It’s good to note that the same documentation linked above goes into further detail about its timing if it interests you.*
+## The LocalDate API 
+Also with `Java 8 SE` came the [LocalDate Class](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html) which is referred to: 
+> A date without a time-zone in the ISO-8601 calendar system, such as `2007-12-03`.
+> `LocalDate` is an immutable date-time object that represents a date, often viewed as year-month-day. Other date fields, such as day-of-year, day-of-week and week-of-year, can also be accessed. For example, the value "2nd October 2007" can be stored in a `LocalDate`.
+
+An important specification to take note of is:
+> This class does not store or represent a time or time zone. Instead, it is a description of the date, as used for birthdays. It cannot represent an instant on the timeline without additional information such as an offset or time zone.
+## Difference & Which one to use 
+#### Issues with the Date/Time API’s 
+Further reading the [LocalDate](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html) class will lead you to “This class is immutable and thread-safe.” This brings us to our first problem.
+- Both the [Date](https://docs.oracle.com/javase/8/docs/api/java/util/Date.html) and [Calender](https://docs.oracle.com/javase/8/docs/api/java/util/Calendar.html) were not thread-safe. If you're a bit confused as to what it means - Threads are a way to implement concurrency with a computer application this allows for multiple tasks to be running concurrently rather than sequentially (If you have a programming background you can see this as `async`). 
```

Comment:
We haven't talked about threads yet, so I don't assume they are ready to talk about thread safety/unsafety.